### PR TITLE
Fix elasticsearch

### PR DESCRIPTION
**Pull request by larskuhtz**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -13,7 +13,7 @@ MINER_PRIVATE_KEY=da81490c7efd5a95398a3846fa57fd17339bdf1b941d102f2d3217ad29785f
 DISABLE_POW_VALIDATION=1
 
 # Replica counts for different node types
-COMMON_NODE_REPLICAS=2
-MINING_NODE_REPLICAS=1
+COMMON_NODE_REPLICAS=1
+MINING_NODE_REPLICAS=0
```
Comment (jwiegley) [inline .env:17]: Should the default environment have no mining nodes?

Comment (DevopsGoth) [reply .env:17]: The mining node does CPU mining that ends up using a lot of resources, I think it should default to 0, the use case for testing mining like that is very rare afaik.

**Review by DevopsGoth — APPROVED** [review]
We can build on these dashboards or accept community contributions over time.

File: .env (hunk 1 above)
Comment (jwiegley) [reply .env:17]: That sound reasonable.

**Review by jwiegley — APPROVED** [review]

**Pull request merged**